static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -133,27 +131,6 @@ define([
                 navigation.init();
             },
 
-            initialiseStickyAdBanner: function () {
-                if (!(config.switches.disableStickyAdBannerOnMobile && detect.getBreakpoint() === 'mobile')
-                    && !config.page.shouldHideAdverts
-                    && config.page.section !== 'childrens-books-site'
-                    && !config.tests.abNewHeaderVariant
-                    && (config.page.hasSuperStickyBanner
-                        || config.page.contentType !== 'Interactive'
-                        && config.page.contentType !== 'Crossword'
-                        && config.page.contentType !== 'Hosted'
-                        && !config.page.isImmersive
-                        && !config.page.isUsMinute
```

Comment:
i don't think this key ever exists, @gtrufitt, its called `isMinuteArticle` client-side?